src/neighbors.cc

```diff
@@ -1503,78 +1503,78 @@ static void
 dump_peers(StoreEntry *sentry, CachePeers *peers)
 {
     char ntoabuf[MAX_IPSTRLEN];
+    PackableStream os(*sentry);
     int i;
 
     if (!peers) {
-        storeAppendPrintf(sentry, "There are no neighbors installed.\n");
+        os << "There are no neighbors installed.\n";
         return;
     }
 
     for (const auto &peer: *peers) {
         const auto e = peer.get();
         assert(e->host != nullptr);
-        storeAppendPrintf(sentry, "\n%-11.11s: %s\n",
-                          neighborTypeStr(e),
-                          e->name);
-        storeAppendPrintf(sentry, "Host       : %s/%d/%d\n",
-                          e->host,
-                          e->http_port,
-                          e->icp.port);
-        storeAppendPrintf(sentry, "Flags      :");
+        os << "\n" << std::setw(11) << std::left <<
+            neighborTypeStr(e) << ": " << e->name << "\n";
+        os << "Host       : " << e->host << '/' << e->http_port << '/' <<
+            e->icp.port << "\n";
+        os << "Flags      :";
+        os.flush();
         dump_peer_options(sentry, e);
 
-        for (i = 0; i < e->n_addresses; ++i) {
-            storeAppendPrintf(sentry, "Address[%d] : %s\n", i,
-                              e->addresses[i].toStr(ntoabuf,MAX_IPSTRLEN) );
-        }
+        for (i = 0; i < e->n_addresses; ++i)
+            os << "Address[" << i << "] : " << e->addresses[i].toStr(ntoabuf, MAX_IPSTRLEN) << "\n";
 
-        storeAppendPrintf(sentry, "Status     : %s\n",
-                          neighborUp(e) ? "Up" : "Down");
-        storeAppendPrintf(sentry, "FETCHES    : %d\n", e->stats.fetches);
-        storeAppendPrintf(sentry, "OPEN CONNS : %d\n", e->stats.conn_open);
-        storeAppendPrintf(sentry, "AVG RTT    : %d msec\n", e->stats.rtt);
+        os << "Status     : " << (neighborUp(e) ? "Up" : "Down") << "\n" <<
+            "FETCHES    : " << std::setw(8) << std::right << e->stats.fetches << "\n" <<
```

Comment:
I recommend going in the opposite direction: Instead of adding/duplicating tricks that attempt to make output "pretty" (and arguably fail to accomplish that in many basic cases), I would drop these attempts and just output `key: value` lines. I do not insist on that change in direction.

BTW, if you would like to make output YAML-compliant, please go ahead. Again, I do not insist on that.

Reply:
For now I'll leave the current format, minus the formatting